dataplane/xtask/src/grpc.rs

```diff
@@ -49,11 +49,11 @@ pub async fn update(opts: Options) -> Result<(), Error> {
 
     let req = Request::new(Targets {
         vip: Some(vip.clone()),
-        target: Some(Target {
+        targets: vec![Target {
             daddr: daddr.into(),
             dport: opts.dport,
             ifindex: Some(opts.ifindex),
-        }),
+        }],
```

Comment:
We talked about this one in Zoom, in this xtask we're populating some "test data" to play around with, it could be helpful to add multiple targets here for testing the new load-balancing behavior. Doesn't necessarily have to happen as a part of this PR though if you're strapped for time, just add a comment/followup.